### ISSUE TITLE
gltbx/tst_ellipsoids.py broken on Python 3

### ISSUE THREAD

**Issue by Anthchirp** [cctbx/cctbx_project#360]
loads of tests still failing on python 3. This one is run by azure:
https://dev.azure.com/cctbx/cctbx_project/_build/results?buildId=184
```
  Standard error:
    Traceback (most recent call last):
      File "/__w/1/modules/cctbx_project/gltbx/tst_ellipsoids.py", line 71, in <module>
        run()
      File "/__w/1/modules/cctbx_project/gltbx/tst_ellipsoids.py", line 56, in run
        exercise_ellipsoid()
      File "/__w/1/modules/cctbx_project/gltbx/tst_ellipsoids.py", line 35, in exercise_ellipsoid
        assert x.transpose() * metrics.inverse() * x > 0
    TypeError: '>' not supported between instances of 'sqr' and 'int'
```

**Comment by rjgildea** [comment]
The issue is that `__gt__` is not defined for `scitbx.matrix.sqr()`. In Python 2 this "works" (with unpredictable results), whereas in Python 3 it now fails with a TypeError:
https://portingguide.readthedocs.io/en/latest/comparisons.html#unorderable-types

The particular line of the test appears to be broken, as the actual value that is apparently being tested is actually `-1`, which definitely is not `> 0`:
```
print(x.transpose() * metrics.inverse() * x)
{{-1.0}}
```

**Comment by rjgildea** [comment]
@luc-j-bourhis I believe you added the relevant line in this commit:
https://github.com/cctbx/cctbx_project/commit/81517a5c7d7720b9532e294e83cb7dff7290f30a#diff-853c6b4b585def216365b44938336e77R33

Can you remember what exactly you were trying to test with this line? As it appears you are asserting that `-1 > 0` 🤔

**Comment by luc-j-bourhis** [comment]
sorry for the delay: I was on holidays! yes, indeed, the inequality should be the other way around. Not sure why I tested that anyway: not related to my ellipsoid code.

**Comment by luc-j-bourhis** [comment]
Fixed by 7a6cb21bcfdca5108470257c6788b0f1d40e83d1
